e2e/cypress/integration/scroll/default_images_collapsed_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Scroll', () => {
+    let testTeam;
+
+    before(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            // # Switch the account settings for the test user to have images collapsed by default
+            cy.apiSaveCollapsePreviewsPreference('true');
+
+            testTeam = team;
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2370 Default images to collapsed', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+
+        cy.get('.post-image').should('be.visible');
+
+        cy.postMessage('{enter}');
+
+        // * Observe image preview is collapsed
+        cy.findByLabelText('file thumbnail').should('not.exist');
+
+        // # Refresh the browser
+        cy.reload();
```

Comment:
"Scroll pop" might be a term we used and understood internally but it refers to a sudden change of height between initial load and final render of an element (or post in this case) causing an observed jank or "pop up") in the UI.
With that, we should compare the rendered height between before and after reload. Open to suggestion but I'm good as suggested below even if it's not that reliable due to some Cypress internals in targeting the element. At least a good starting point moving forward. 
```suggestion
        // # Save height of the last post
        cy.getLastPost().then((lastPost) => {
            cy.wrap(parseInt(lastPost[0].clientHeight, 10)).as('lastPostHeight');
        });

        // # Refresh the browser
        cy.reload();

        // * Verify that the last post is still the same after reload
        cy.getLastPost().then((lastPost) => {
            cy.get('@lastPostHeight').then((lastPostHeight) => {
                expect(parseInt(lastPost[0].clientHeight, 10)).to.equal(lastPostHeight);
            });
        });
```